packages/client/src/components/Actions/components/ActionRequired.js

```diff
@@ -3,15 +3,9 @@ import NFTView from "./NFTView";
 import TokenView from "./TokenView";
 import OwnerView from "./OwnerView";
 import ThresholdView from "./ThresholdView";
-import Confirmations from "./Confirmations";
+import Confirmations from "components/Confirmations";
 
-const ActionRequired = ({
-  safeData,
-  actionView,
-  confirmations,
-  onReject,
-  onApprove,
-}) => {
+const ActionRequired = ({ safeData, actionView, confirmations, onReject, onApprove }) => {
```

Comment:
@mannynotfound  please help cross check the formatting, seems the files are committed with a different format, which will result unrelated diff in PR review.

Reply:
@amyliu6 this is happening in my editor since our prettier rules were updated - I think its expected we use this setting so any changes to old files will have unrelated prettier changes:

![image](https://user-images.githubusercontent.com/15314629/193113314-aa046bbe-5f48-4329-89ad-54a2739f08b7.png)

I agree it makes PR a bit harder to reason about but hopefully its just a growing pain to a better future 😅

Reply:
@amyliu6 actually I was just on a new editor that was missing https://github.com/prettier/prettier-vscode#prettierrequireconfig-default-false 😓 I've updated PR now so format should be the same.

Reply:
cool. we also have pre-commit hook to auto format during commit, curious why that didn't kick-in? 🤔